### PR TITLE
Feat : Added result Page

### PR DESCRIPTION
**Pull request by SaurabhChawda**
Added Static Result page.
It will contain:-
- Quiz total score 
- Total Questions with correct Answers.
Screenshot (328)

### CODE REVIEW

**Review by tusharkandpal — APPROVED** [review]
looks good so far! 👍🏻

File: src/Pages/Result-Page/result.css [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  background-color: var(--primary-color);
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+}
+
+/* Results Header Start Here */
+.result--header {
+  display: flex;
+  justify-content: center;
```
Comment (sushil-mod) [inline src/Pages/Result-Page/result.css:13]: this is correct , if possible you can try this . In both the classes have same 2 properties you can  use result-header classname in above with result-page and remove these 2 properies from result-page

**Review by sushil-mod — APPROVED** [review]
just a suggestion . your result page Ui alos looks good . rest code is nice

**Comment by SaurabhChawda** [comment]
> looks good so far! 👍🏻

ThankYou

File: src/Pages/Result-Page/result.css (hunk 1 above)
Comment (SaurabhChawda) [reply src/Pages/Result-Page/result.css:13]: Okay, I will update,ThankYou

**Pull request merged**